vss_invoice_mgmt/src/pages/business/pages/masters/productmaster/pages/ProductMaster.jsx

```diff
@@ -30,14 +42,46 @@ const product_ValidationSchema = Yup.object().shape({
     .required("Please enter product CGST")
     .positive("Please enter a positive number"),
```

Comment:
Add validation for IGST, SGST, CGST . These values should be not greater than 28.

Reply:
@roiveredritesh can you give me some examples for IGST,SGST,CGST

Reply:
@roiveredritesh Since as per my understanding. The IGST, CGST, SGST should be of of number type & we can't have a number of 28 digit & if we keep then it's going to be a string.

Reply:
@neha199704 You were partially right. IGST, CGST, SGST should be number type but should not be greater than 28. It means value should be between 0-28, not more than 28. 28 is value not digit length.